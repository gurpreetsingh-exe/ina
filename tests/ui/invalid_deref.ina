// COMPILE_FAIL

fn main() {
    let a = *20;
}
