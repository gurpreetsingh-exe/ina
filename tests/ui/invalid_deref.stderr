error: invalid dereference
   --> invalid_deref.ray:4:14
    |
 LL |     let a = *20;
    |              ^^ `int` cannot be dereferenced

